stdlib/src/socket.rs

```diff
@@ -82,12 +82,9 @@ mod _socket {
 
     #[pyattr(once)]
```

Comment:
When the function is guaranteed to return same object everytime, we don't need `once`.

```suggestion
    #[pyattr]
```